src/Behat/Behat/Context/Cli/InteractiveContextIdentifier.php

```diff
@@ -16,7 +16,7 @@
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Console\Question\ChoiceQuestion;
-use Symfony\Component\Translation\TranslatorInterface;
+use Symfony\Contracts\Translation\TranslatorInterface;
```

Comment:
I think it would be better to rebase this branch or build it on top of @flug's PR #1247 he did already implemented a BC Layer for the TranslatorInterface and EventDispatcher. Or maybe @flug can tell us what is missing in his PR.

Reply:
Well I'm waiting to know if behat will continue to be backward compatible with the old version because contract works it seems to me that from version 7.1 onwards and it refers to this other pull request https://github.com/Behat/Behat/pull/1249

Reply:
Sorry I thought I'd replied clearly; while we might want to consider dropping support for old versions let's have a separate conversation about it, it shouldn't stop us making symfony 5 work

Reply:
And yes @japicoder please look at the work @flug has done to avoid overlap. Maybe one of these PRs should rebase onto the other?

Reply:
Hi, I think that what makes more sense is to take what @flug did already, as he published first the PR, and update it to the latest Behat master branch because I see it is one or two commits below.

@ciaranmcnulty can you guide me how can I rebase safely? I never did it before in Github.